api/grpcserver/v2alpha1/activation_test.go

```diff
@@ -3,8 +3,10 @@ package v2alpha1
 import (
 	"context"
 	"errors"
+	"golang.org/x/exp/rand"
```

Comment:
Please use `math/rand` if possible - the same functionality should be available there.